src/test/java/core/basesyntax/service/impl/WriterImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.impl;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterImplTest {
+    private static WriterImpl writer;
+
+    @BeforeClass
+    public static void beforeClass() {
+        writer = new WriterImpl();
+    }
+
+    @Test
+    public void write_correctPath_ok() {
```

Comment:
```suggestion
    public void writeToFile_correctPath_ok() {
```
давай посмотрим все методы во всех тестовых классах и приведем их названия к одному виду

методкоторыйтестируем_условие_результат